static/include/components/base-control.md

```diff
@@ -0,0 +1,5 @@
+## Test the base
+
+After you configure the base, go to the [control](/manage/fleet/robots/#control) tab and expand the base component in order to enable keyboard or [discrete](https://docs.viam.com/try-viam/try-viam-tutorial/#discrete-movement-control) control over your robot's movement. The Live Feed section contains buttons to toggle cameras on or off and to arrange how you view the camera feeds.
```

Comment:
Just a few small things!

- For UI elements in the Viam App, we use **bold** to signify a UI element the user will see in the App UI.
- We also keep each sentence on its own separate line to help with reviewing and to keep code width reasonable (don't worry: it still renders as a sentence within the same paragraph, unless you put an entire blank newline in between sentences).
- Not every base will feature the **Live Feeds** section, it depends on whether a camera is configured for the robot / base.

All together, this paragraph might resemble something like:

```suggestion
After you configure the base, go to the [**Control**](/manage/fleet/robots/#control) tab and expand the base component to view the controls to enable keyboard or [discrete](https://docs.viam.com/try-viam/try-viam-tutorial/#discrete-movement-control) control over your robot's movement.
If your base makes use of a [camera](/components/camera/) component, the **Control** tab for your base will also display a **Live Feed** section which contains buttons to toggle cameras on or off, and to arrange how you view the camera feeds.
```